pkg/ppf/crew.go

```diff
@@ -0,0 +1,128 @@
+package ppf
+
+import (
+	"encoding/csv"
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/diagmatrix/maybe-better-maybe-worse/pkg/utils"
+)
+
+// MemberType represents the different kinds of crew members of the shooting plan
+type MemberType int
+
+// ErrInvalidMemberType is returned when the member type is invalid
+var ErrInvalidMemberType = errors.New("invalid member type")
+
+const (
+	Actor MemberType = iota
+	Crew
+)
+
+// MemberTypeFromString converts a string to a MemberType
+// It takes a string as an argument
```

Comment:
¿"it takes" y "it returns" es el formato docstring estándar en Go?

Reply:
Por lo que vi [aquí](https://go.dev/doc/comment), no es mala praxis

Reply:
Pues por lo que yo veo [aquí](https://go.dev/doc/comment#func), lo que dice es:

> Named parameters and results can be referred to directly in the comment, without any special syntax like backquotes

Por lo que quizás deberías usar explícitamente el nombre de los argumentos.